calculator/src/main/java/edu/nextstep/camp/calculator/viewmodel/CalculatorViewModel.kt

```diff
@@ -26,13 +23,12 @@ class CalculatorViewModel(
     private val _viewTypeEvent = SingleLiveEvent<CalculatorViewType>()
     val viewTypeEvent: LiveData<CalculatorViewType> get() = _viewTypeEvent
 
+    private val _memoriesEvent = SingleLiveEvent<List<Calculation>?>()
+    val memoriesEvent: LiveData<List<Calculation>?> get() = _memoriesEvent
```

Comment:
```suggestion
    private val _memoriesEvent = SingleLiveEvent<List<Calculation>>()
    val memoriesEvent: LiveData<List<Calculation>> get() = _memoriesEvent
```
컬렉션은 null 대신 빈 리스트를 활용해주시면 좋을 것 같아요